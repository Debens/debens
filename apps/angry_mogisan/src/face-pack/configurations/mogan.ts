import { BundledProfileName } from '../../face-book/bundled/model';
import { FacePack, FaceType } from '../model';

export const mogan: FacePack = {
    profiles: [
        {
            type: FaceType.Bundled,
            name: BundledProfileName.Morgan,
        },
    ],
};
