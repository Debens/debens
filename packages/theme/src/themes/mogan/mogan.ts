import type { Theme } from '../../model';
import { FontFamily, FontSize, FontWeight, LineHeight } from '../../tokens/typography';
import { Color } from '../../values/colors';

export const m√≥gan: Theme = {
    space: {
        none: 0,
        auto: 'auto',
        miniscule: 2,
        xsmall: 4,
        small: 8,
        medium: 16,
        large: 24,
        xlarge: 32,
    },
    radii: {
        none: 0,
        xsmall: 2,
        small: 4,
        medium: 8,
        large: 16,
        xlarge: 24,
    },
    colors: {
        $transparent: Color.None,

        // background
        '$background-primary': Color.Magenta30,
        '$background-active': Color.Magenta50,
        '$background-brand': Color.Magenta60,

        // layer
        '$layer-01': Color.Magenta20,
        '$layer-hover-01': Color.Magenta10,
        '$layer-active-01': Color.Magenta30,
        '$layer-selected-01': Color.Gray20,
        '$layer-02': Color.White,
        '$layer-hover-02': Color.Gray10Hover,
        '$layer-active-02': Color.Gray30,
        '$layer-selected-02': Color.Gray20,
        '$layer-03': Color.Gray10,
        '$layer-hover-03': Color.Gray10Hover,
        '$layer-active-03': Color.Gray30,
        '$layer-selected-03': Color.Gray20,
        '$layer-disabled': Color.Gray50,

        // text
        '$text-primary': Color.Magenta60,
        '$text-secondary': Color.White,
        '$text-on-color': Color.Magenta20,
        '$text-placeholder': Color.Gray40,
        '$text-disabled': Color.Gray30,
        '$text-error': Color.Red60,

        // button
        '$button-primary': Color.Magenta60,
        '$button-primary-hover': Color.Magenta50,
        '$button-primary-active': Color.Magenta80,
        '$button-secondary': Color.Magenta20,
        '$button-secondary-hover': Color.Magenta10,
        '$button-secondary-active': Color.Magenta30,
        '$button-danger': Color.Red60,
        '$button-danger-hover': Color.Red60Hover,
        '$button-danger-active': Color.Red80,
        '$button-disabled': Color.Gray30,

        '$field-01': Color.Magenta20,
        '$field-error-01': Color.Red20,
        '$field-selected-01': Color.Magenta60,

        // other
        $ghost: Color.Gray30,
    },
    borderWidths: {
        none: 0,
        small: 1,
    },
    typography: {
        $body: {
            fontFamily: FontFamily.Unknown,
            fontSize: FontSize.Small,
            lineHeight: LineHeight.Small,
            fontWeight: FontWeight.Light,
        },
        $heading: {
            fontFamily: FontFamily.Unknown,
            fontSize: FontSize.Large,
            lineHeight: LineHeight.Large,
            fontWeight: FontWeight.Normal,
        },
    },
    contextual: {},
};
