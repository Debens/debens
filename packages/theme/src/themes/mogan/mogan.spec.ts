import { mógan } from './mogan';

describe('mógan theme', () => {
    it('then should not have any unexpected changes', () => {
        expect(mógan).toMatchSnapshot();
    });
});
