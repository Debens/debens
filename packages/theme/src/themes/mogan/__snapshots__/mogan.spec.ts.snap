// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`m√≥gan theme then should not have any unexpected changes 1`] = `
Object {
  "borderWidths": Object {
    "none": 0,
    "small": 1,
  },
  "colors": Object {
    "$background-active": "#ee5396",
    "$background-brand": "#d02670",
    "$background-primary": "#ffafd2",
    "$button-danger": "#da1e28",
    "$button-danger-active": "#750e13",
    "$button-danger-hover": "#ba1b23",
    "$button-disabled": "#c6c6c6",
    "$button-primary": "#d02670",
    "$button-primary-active": "#740937",
    "$button-primary-hover": "#ee5396",
    "$button-secondary": "#ffd6e8",
    "$button-secondary-active": "#ffafd2",
    "$button-secondary-hover": "#fff0f7",
    "$field-01": "#ffd6e8",
    "$field-error-01": "#ffd7d9",
    "$field-selected-01": "#d02670",
    "$ghost": "#c6c6c6",
    "$layer-01": "#ffd6e8",
    "$layer-02": "#ffffff",
    "$layer-03": "#f4f4f4",
    "$layer-active-01": "#ffafd2",
    "$layer-active-02": "#c6c6c6",
    "$layer-active-03": "#c6c6c6",
    "$layer-disabled": "#8d8d8d",
    "$layer-hover-01": "#fff0f7",
    "$layer-hover-02": "#e5e5e5",
    "$layer-hover-03": "#e5e5e5",
    "$layer-selected-01": "#e0e0e0",
    "$layer-selected-02": "#e0e0e0",
    "$layer-selected-03": "#e0e0e0",
    "$text-disabled": "#c6c6c6",
    "$text-error": "#da1e28",
    "$text-on-color": "#ffd6e8",
    "$text-placeholder": "#a8a8a8",
    "$text-primary": "#d02670",
    "$text-secondary": "#ffffff",
    "$transparent": "transparent",
  },
  "radii": Object {
    "large": 16,
    "medium": 8,
    "none": 0,
    "small": 4,
    "xlarge": 24,
    "xsmall": 2,
  },
  "space": Object {
    "auto": "auto",
    "large": 24,
    "medium": 16,
    "miniscule": 2,
    "none": 0,
    "small": 8,
    "xlarge": 32,
    "xsmall": 4,
  },
  "typography": Object {
    "$body": Object {
      "fontFamily": "",
      "fontSize": 12,
      "fontWeight": 400,
      "lineHeight": "12px",
    },
    "$heading": Object {
      "fontFamily": "",
      "fontSize": 24,
      "fontWeight": 600,
      "lineHeight": "24px",
    },
  },
}
`;
